infra/reachability-metadata/src/main/resources/META-INF/native-image/com.github.ben-manes.caffeine/caffeine/2.9.3/reflect-config.json

```diff
@@ -99,19 +99,16 @@
   "name":"com.github.benmanes.caffeine.cache.StripedBuffer",
   "fields":[{"name":"tableBusy"}]
 },
+{
+  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.NodeFactory"},
+  "name":"com.github.benmanes.caffeine.cache.PDWMS",
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
 {
   "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.StripedBuffer"},
   "name":"java.lang.Thread",
   "fields":[{"name":"threadLocalRandomProbe"}]
 },
-{
-  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.LocalLoadingCache"},
-  "name":"org.apache.shardingsphere.infra.parser.cache.SQLStatementCacheLoader"
-},
-{
-  "condition":{"typeReachable":"com.github.benmanes.caffeine.cache.LocalLoadingCache"},
-  "name":"org.apache.shardingsphere.sql.parser.core.database.cache.ParseTreeCacheLoader"
-},
```

Comment:
- @strongduanmu I noticed that this was introduced via #32633 . But the question is, these two JSONs are actually already located in `infra/reachability-metadata/src/main/resources/META-INF/native-image/org.apache.shardingsphere/generated-reachability-metadata/reflect-config.json`, does this affect any unit tests?

Reply:
Hi @linghengqian, there reachability metadata are added for SphereEx commercial features. Since comments cannot be added to JSON files, it is difficult to mark the differences between open source and commercial with some comments, so I submitted them to the open source version.

Reply:
> Hi @linghengqian, there reachability metadata are added for SphereEx commercial features. Since comments cannot be added to JSON files, it is difficult to mark the differences between open source and commercial with some comments, so I submitted them to the open source version.

- It's still strange that the JSON entries for `org.apache.shardingsphere.**` are placed here, since those classes don't belong to https://github.com/ben-manes/caffeine . This could actually be misleading to downstream.

- As for adding JSON comments, this needs to wait for the nightly CI involved in #33025 to be fixed. The JSON comments support mentioned in https://www.graalvm.org/docs/reference-manual/native-image/assets/reachability-metadata-schema-v1.0.0.json and https://github.com/oracle/graal/issues/8543 is only in GraalVM CE For JDK 23 and downstream releases.

```json
{
  "reflection": [
    {
      "type": "org.apache.shardingsphere.infra.yaml.config.pojo.mode.YamlModeConfiguration",
      "methods": [
        {
          "name": "setType",
          "parameterTypes": ["java.lang.String"] 
        }
      ],
      "allDeclaredFields":true,
      "queryAllPublicMethods":true
    }
  ],
  "resources": [
    {
      "glob": "META-INF/services/org.apache.shardingsphere/**"
    }
  ],
  "bundles": [
    {
      "name": "com.microsoft.sqlserver.jdbc.SQLServerResource",
      "locales": ["zh-CN"]
    }
  ],
  "reason": [
      "test json comment first",
      "test json comment second",
      "test json comment third"
  ],
  "condition": {
     "typeReached": "org.apache.shardingsphere.infra.spi.type.ordered.OrderedSPILoader"
  }
}
```

- I assume those JSONs could be added to somewhere like the private directory `infra/reachability-metadata/src/main/resources/META-INF/native-image/com.example.private/private-maven-module` or `private-maven-module/src/main/resources/META-INF/native-image/com.example.private/private-maven-module`. I don't know anything about the unit tests involved downstream.